.gitignore

```diff
@@ -8,6 +8,11 @@ node_modules/
 # Vagrant files
 .vagrant
 
+# Ansible
+./group_vars
+hosts
```

Comment:
Seems like `hosts` file needs to exist or Ansible won't even start.

We might very well keep some defaults like:
```
[local]
127.0.0.1

[production]
hitchwiki.org

[beta]
beta.hitchwiki.org
```
...but I suppose it needs to allow adding custom targets during development? — same trick as with settings.yml?

Reply:
hosts is defined in ansible.cfg, to get rid of hosts, delete both.

To make your sample work i need to swap:
- `group_vars/all` - is available to all and currently has
- `group_vars/hitchwiki` links to configs/settings.yml and is only available to hosts in the group [hitchwiki]

My hosts has
```
[hitchwiki]
vagrant ansible_ssh_host=127.0.0.1 ansible_user=ubuntu ansible_ssh_port=2222 ansible_ssh_private_key_file=/home/hitchwiki/src/hitchwiki/.vagrant/machines/hitchwiki/virtualbox/private_key
hw-dev ansible_ssh_host=... ansible_user=hitchwiki
```

To set variables per host use `host_vars/name`. My `host_vars/vagrant` has "apt_proxy: pi".